plugins/source/heroku/codegen/recipes/recipes.go

```diff
@@ -31,6 +32,138 @@ type Resource struct {
 	PrimaryKey string
 }
 
+type AddOnWebhookDelivery struct {
+	heroku.AddOnWebhookDeliveryInfoResult
+}
+
+type AddOnWebhookEvent struct {
+	heroku.AddOnWebhookEventInfoResult
+}
+
+type AppWebhook struct {
+	heroku.AppWebhookInfoResult
+}
+
+// Element of PipelineBuildListResult
+type PipelineBuild struct {
```

Comment:
If there's a better way to do this, please let me know

Reply:
`make(PipelineBuildListResult, 1)[0]`?